### ISSUE TITLE
Can't printPit

### ISSUE THREAD

**Issue by erfanium** [Samsung-Loki/Thor#17]
## Description
Can't printPit

## Reproduction
![image](https://github.com/Samsung-Loki/Thor/assets/47688578/d3ee5b06-252d-4bc6-8ae0-1b4d7a5cb9bd)

## Expected behaviour
Describe what you expected to happen.

## Hardware and software
 Kernel: 6.2.0-39-generic

## Additional context
I can't get pit details even with odin for windows or Heimdall, that's why i tried Thor. I'm not sure it's because of my device, or not. I have a Samsung A5 2017 that only boots into downloading mode. I can't boot into recovery or normal mode

**Comment by thongass000** [comment]
@erfanium hey have you found a workaround? I have the same problem with Odin, Heimdall and Thor and I have a Samsung A7 2017 only booting into download mode.

**Comment by erfanium** [comment]
@thongass000 No. seams this problem is related to hardware storge failure. i simply gave up on my phone
